### ISSUE TITLE
[important] Generator?

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-statements#30]
... thinking from users' perspectives... maybe "Generator" isn't the best word for it?

Any input related to translation and non-native speakers?

/me brainstorms (=some bad but might help others think of better :-)...
* Accessibility Statement Tool
* Accessibility Statement Content Tool
* Accessibility Statement Creator Tool
* Accessibility Statement Authoring Tool
...

(some more thoughts on title are in [27 Sept minutes](https://www.w3.org/2018/09/27-eo-minutes.html) )

**Comment by rvantonisse** [comment]
I agree there must at least be some consistency in titles, navigation, breadcrumb and mentioned tool in contexts.
I believe the initial project name is "Accessibility Statement Generator"? 

> D3.1: Accessibility Statement Generator …
[WAI Tools Project deliverables](https://www.w3.org/WAI/Tools/#deliverables)

And for page naming this would be good: 

- "Accessibility Statement Generator"  `index`
- "Create Accessibility Statement" `create(-accessibility-statement)`
- "Accessibility Statement Preview" `(accessibility-statement-)preview`

**Comment by slhenry** [comment]
> D3.1: Accessibility Statement Generator …
> [WAI Tools Project deliverables](https://www.w3.org/WAI/Tools/#deliverables)

I'm pretty sure that "Accessibility Statement Generator" can be considered a "working title" and we do not need to use it for the final resource. (@nitedog can confirm that)

[Related issue: [pages titles, nav, button](https://github.com/w3c/wai-statements/issues/).]

**Comment by nitedog** [comment]
Yes, the title can be changed if needed.
